File: Java/src/Ejercicio9.java
import java.util.Scanner;

public class Ejercicio9 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Introduce un nuemero");
        int n = sc.nextInt();
        int aux = 0;
        if(n%2!=0){
            for (int i = 1; i <= n; i++){
                if (n%i==0){
                    aux++;
                }
            }
            if (aux >= 3){
                System.out.println("El numero no es primo");
            }else System.out.println("El numero es primo");
        }else System.out.println("El numero no es primo");
    }
}

File: Kotlin/src/Ejercicio9.kt
fun main(){
    println("Introduce un número")
    val num = readln().toInt()
    var aux = 0
    if (num % 2 != 0){
        for (i in 1..num){
            if (num % i == 0){
                aux++
            }
        }
        if (aux >= 3){
            println("El numero no es primo")
        }else println("El numero es primo")
    }else println("El numero no es primo")
}